packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -94,6 +75,32 @@ export function activate(context: vscode.ExtensionContext) {
     debugConfigProvider,
     checkpointsView
   );
+
+  console.log('in activate, getting line breakpoint info');
+  if (!await retrieveLineBreakpointInfo()) {
+    console.log(
+      'in activate, did not retrieve line breakpoint info, returning'
+    );
+    // the error was already reported in the function, return
+    return;
+  }
+  console.log('in activate, successfully retrieved line breakpoint info');
+
+  console.log('in activate, activating request service');
+  if (!await checkpointService.activateRequestService()) {
+    console.log('in activate, activating request service failed, returning');
```

Comment:
@JimSuplizio - Can we remove this `console.log`?

CC: @DatGuyJonathan , @lcampos